src/libs/actions/IOU.ts

```diff
@@ -1432,7 +1471,7 @@ function createSplitsAndOnyxData(participants, currentUserLogin, currentUserAcco
         {
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${CONST.IOU.OPTIMISTIC_TRANSACTION_ID}`,
-            value: null,
+            value: {},
```

Comment:
This will not work, it's the same as not doing anything, which will break the functionality - I'm pretty sure in cases like this we should change `onyxMethod` to `Onyx.METHOD.SET` and keep `value: null`